Core/MIPS/ARM64/Arm64Jit.cpp

```diff
@@ -725,6 +725,9 @@ void Arm64Jit::UpdateRoundingMode(u32 fcr31) {
 // though, as we need to have the SUBS flag set in the end. So with block linking in the mix,
 // I don't think this gives us that much benefit.
 void Arm64Jit::WriteExit(u32 destination, int exit_num) {
+	// TODO: Check destination is valid and trigger exception.
```

Comment:
Note that sometimes code will have untaken invalid exits, such as jumps to zero and such.  Usually they are breaks, though, but not always.

-[Unknown]

Reply:
Ah, good point. Will sneak in a comment update later.